### ISSUE TITLE
Total Valley Bottom Feature Class Metrics

### ISSUE THREAD

**Issue by philipbaileynar** [Riverscapes/riverscapes-tools#430]
The following metrics should be calculated and stored on the `vbet_full` feature class.

The ID column below is purely for communication. It's not meant to be stored in the feature class etc.

|ID|Attribute Column Name|Units|Method|
|---|---|---|---|
|A|valley_bottom_ha|hectares|The sum of areas for all `vbet_full` features that have the same levelPath|
|B|active_valley_bottom_ha|hectares|The sum of areas for all `vbet_ia` features that have the same levelPath|
|C|active_channel_ha|hectares|Intersect of the channel area feature class with `vbet_full`|
|D|active_floodplain_ha|hectares|The sum of the areas for all `active_floodplain` features that have the same levelPath|
|E|inactive_floodplain_ha|hectares|The sum of areas for all `inactive_floodplain` features that have the same levelPath|
|F|active_valley_bottom_pc|percent|100 * B/A|
|G|active_channel_pc|percent|100 * C/A|
|H|active_floodplain_pc|percent|100 * D/A|
|I|inactive_floodplain_pc|percent|100 *E/A|

Note how levelPath is used. to identify the features in each feature class that correspond to parent record in `vbet_full` that gets the metrics for each feature.

**Comment by joewheaton** [comment]
@philipbaileynar this is really cool. I read each method for A - I and can confirm those appear correct. Can't wait to check it out. Thanks! @lauren-herbine and @shelbysawyer FYI... this relates to #395 and #108.

**Comment by joewheaton** [comment]
@philipbaileynar we also discussed counting the number of `inactive_floodplain` polygons/features in the same levelPath, and counting the number of `active_floodplain` polygons/features and adding those as metrics to the `vbet_full` attribute table. Is that still possible?

If so, is this the spec?


ID | Attribute Column Name | Units | Method
-- | -- | -- | --
J | cnt_active_fp_units | count (int)| The total number of `active_floodplain` features that have the same levelPath 
K | cnt_inactive_fp_units | count (int)| The total number of `inactive_floodplain` features that have the same levelPath

**Comment by KellyMWhitehead** [comment]
@philipbaileynar, @joewheaton 

Please review [this project](https://northarrowresearchlabs.github.io/riverscapes-staging/#/Anabranch/a221138b-668b-4d65-b15d-f4bf1679374b) for the new vbet area metrics.

**Comment by joewheaton** [comment]
Awesome @KellyMWhitehead. This is really cool. Just taking a look here:

https://youtu.be/SC3dN4xU3d8
[![Video Link](http://img.youtube.com/vi/SC3dN4xU3d8/0.jpg)](http://www.youtube.com/watch?v=SC3dN4xU3d8 "")

- [ ] For all the hectare fields only keep 5 significant figures past the decimal 
- [ ] For the percentage fields only keep 2 significant figures.
- [ ] Why is there a `area_ha` and a `valley_bottom_ha`? They appear to be the same. Just keep `valley_bottom_ha`
- [ ] Can we do alias names?

**Comment by KellyMWhitehead** [comment]
1. I found a way to round the metric values. I will apply this in future vbet runs.
2.  `area_ha` is the area of the *feature* in hectares, `valley_bottom_ha` is the total area of the valley bottom grouped by `vbet_level_path`. 
3. Alias names: I tried a few things within the geopackage, but haven't found anything that works yet. I think a better way is to look into saving the field alias in the .lyr and qgis layer files?
